docs/05/d.md

```diff
@@ -34,7 +34,7 @@ Here's [the typeclass contract for `MonadFilter`][MonadFilterSource]:
 }
 ```
 
-This enables a derivative function/operator `filter` and `filterM`:
+This enables the derivative functions/operators `filter` and `filterM`:
```

Comment:
By know, you should've noted that I have little opinion about number and articles. I can hardly distinguish eating chicken from eating a chicken.